frontend/components/SendCords.vue

```diff
@@ -0,0 +1,110 @@
+<!-- this is file is used to take input from the user and format it correctly and send to the backend, algorithm and map is gotten from different file, but is add to the request here.-->
+<template>
+  <div id="sendCords">
+    <!-- Creates 4 inputs field for coordinates, displays first 2 coordinates recived-->
+
+    Start X <br /><input v-model="coordinates.start.x" /><br />
+    Start Y <br /><input v-model="coordinates.start.y" /><br />
+    End X <br /><input v-model="coordinates.stop.x" /> <br />
+    End Y <br /><input v-model="coordinates.stop.y" /> <br />
+
+    <button v-on:click="submitPoints">Send</button>
+  </div>
+</template>
+<script>
+import axios from "axios";
+//used to import data from other components
+import { store, mutations } from "../store.js";
+import { getRoute } from "route";
+
+export default {
+  computed: {
+    tester() {
+      return store.tester;
+    }
+  },
+  data: function() {
+    return {
+      coordinates: {
+        //coordinates to be sent
+        start: { x: null, y: null },
+        stop: { x: null, y: null },
+        map_id: null,
+        algorithm: {
+          name: null,
+          version: null
+        }
+      },
+      job_token: {},
+      display: {
+        data: {
+          points: []
+        }
+      },
+      messageSent: false,
+      map_id: null
+    };
+  },
+
+  computed: {
+    selected_algorithms() {
+      return store.selected_algorithms;
+    }
+  },
+  methods: {
+    submitPoints: async function() {
+      //Convert inputs coords to ints
+      this.coordinates.start.x = parseInt(this.coordinates.start.x);
```

Comment:
Use temporary variables instead of allowing this.coordinates to have one of two different types?